tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php

```diff
@@ -722,6 +722,22 @@ public function createdOrderShouldHaveFreeShipping(string $reference)
         }
     }
 
+    /**
+     * @Then order :reference should be a gift with message :message
```

Comment:
```suggestion
     * @Then the order :reference should be a gift with the message :message
```

Reply:
I'm not sure this is relevant 🤔 

First, we have multiple other Behat steps where we do not use `the`: https://github.com/PrestaShop/PrestaShop/blob/develop/tests/Integration/Behaviour/Features/Context/Domain/OrderFeatureContext.php#L377
So I suggest sticking to this style to keep the scenarii way of being written consistent 😉 

Second, I think the sentence "Order reference 17636 should be a gift with message 'I love you'" is syntaxically right. I dont think there is a need to add `the` before every word. If there was no reference, so "which order are we talking about" would be a valid question, `the` would be useful to identify the order.

For example following sentence is OK: "Core Developer Progi1984 should be very efficient with tool Prestonbot". You don't need to turn it into "The Core Developer Progi1984 should be very efficient with the tool Prestonbot" 😉

Reply:
I agree but we try to improve them bit by bit, it's not blocking but each time @Progi1984 was arguing I prefered not to discuss it as it takes more time and energy than making the change 😛